Homework 4/lesson_4.py

```diff
@@ -0,0 +1,73 @@
+from database_handler import execute_query
+from flask import Flask
+from webargs.flaskparser import use_kwargs
+from validator import order_price_validator, track_validator
+
+app = Flask(__name__)
+
+
+@app.route("/order-price")
+@use_kwargs(order_price_validator, location='query')
+def order_price(country) -> str:
+    """The function displays total sales value from chinook.db.
+    If the Country value is specified, the function returns the total value for that country."""
+
+    if country:
```

Comment:
this isn`t good if, use query and if you have country just add where BillingCountry=...